src/bundle/Resources/public/scss/ui/modules/universal-discovery/_collapsible.scss

```diff
@@ -0,0 +1,100 @@
+.c-collapsible {
+    $self: &;
+
+    & + .c-filters-row,
+    & + .c-filters__row {
+        border-top: calculateRem(1px) solid $ibexa-color-light;
+    }
+
+    &--hidden {
+        #{$self} {
+            &__title {
+                box-shadow: none;
+                border-bottom-color: transparent;
+
+                &:before {
+                    transform: rotate(135deg);
+                }
+
+                &:after {
+                    transform: rotate(-135deg);
+                }
+            }
+
+            &__content {
+                transform: scaleY(0);
+                opacity: 0;
+                height: 0;
+            }
+        }
+    }
+
+    &__title {
+        position: relative;
+        cursor: pointer;
+        padding: 0 calculateRem(24px);
+        font-size: $ibexa-text-font-size-medium;
+        font-weight: 600;
+        min-height: calculateRem(40px);
+        display: flex;
+        align-items: center;
+        border-style: solid;
+        border-color: $ibexa-color-light;
+        border-width: calculateRem(1px) 0;
+        box-shadow: calculateRem(4px) calculateRem(22px) calculateRem(47px) 0 rgba($ibexa-color-info, 0.05);
+
+        &::before,
+        &::after {
```

Comment:
maybe we could do mixin from it? As I guess that's an arrow, already used in filters in two other places (also in theory arrow is also in dropdown, but in different direction, so probably unable to mixinise...)